Bhagyashri/html/account_opening_form.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+ <head>
+ <title> Acount Opening Form</title>
+ </head>
+  <body>
+   <form>
+    <h3>Account Opening Form</h3>
+	<h4>Full Name * </h4>
+<select>
+	<option> MR. </option>
+	<option> MRS.</option>
+	<option> Ms.</option>	
+</select>
+<input type="text" maxlength="20" placeholder="Enter First Name"/>
+<input type="text" maxlength="30" placeholder="Enter Last Name"/>
+<table>
+<tr>
+  <td><h4>Account Type * </h4></td>
+  <td>
+    <select>
+		<option> Please Select </option>
+		<option> Saving </option>
+		<option> Salary </option>
+		<option> Current </option>
+		<option> Other </option>			
+   </select>
+ </td>
+</tr>
+<tr>
+  <td><h4>ID Card Number<h4></td>
+  <td>
+  <input type="text" placeholder="ex 123" maxlength="16"/>
+   </td>
+</tr>
+<tr>
+  <td><h4>Gender</h4></td>
+  <td>
+   <input type="radio" name=gender/> Male
+   <input type="radio" name=gender/> Female
+  </td>
+</tr>
+<tr>
+  <td><h4>Email ID</h4></td>
+  <td>
+  <input type="text" Placeholder="Enter Email ID"/>
```

Comment:
Add value attribute with emapty value. also make sure to add the unique name attributes to the element.